client/src/components/Carousel.css

```diff
@@ -0,0 +1,48 @@
+.carousel{
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: white;
+    padding-top: 15px;
+}
+
+.image{
+    width: 345px;
+    height: 250px;
+    box-shadow: 0px 0px 5px 0px rgb(85, 85, 85);
+}
+
+.right-arrow{
+    position: absolute;
+    top: 48%;
+    right: 22px;
+    font-size: 1.7rem;
+    color: white;
+    user-select: none;
```

Comment:
very interesting, never seen this css property :)

Reply:
its on an imported icon. It won't be selected when you accidentally double click or mark. so it don't behaves like a text. as far as I understood this :)